pkg/virtual/workspaces/registry/rest.go

```diff
@@ -674,8 +465,7 @@ func (s *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 
 	// Update the cluster role bindings with the new workspace internal and pretty names,
```

Comment:
Same here, we can do this in a single Create instead of Create+Update?

Reply:
Additional cleanup done in commit ab6cf45a76494764713e1bd66e27413c22c7f29f